elephant/spike_train_correlation.py

```diff
@@ -163,6 +188,13 @@ def __calculate_correlation_or_covariance(binned_sts, binary, corrcoef_norm):
         Use normalization factor for the correlation coefficient rather than
         for the covariance.
     '''
+    sparse = binned_sts.to_sparse_array()
+    row_counts = sparse.getnnz(1)
+    if row_counts.min() == 0:
+        warnings.warn('Detected rows without spikes.')
```

Comment:
I would suggest to phrase the warning more on the conceptual level, e.g.,
"Detected empty spiketrains (rows) in argument `binned_sts`"